superset/assets/src/dashboard/util/propShapes.jsx

```diff
@@ -59,8 +59,8 @@ export const slicePropShape = PropTypes.shape({
   datasource: PropTypes.string,
   datasource_name: PropTypes.string,
   datasource_link: PropTypes.string,
-  changedOn: PropTypes.number,
-  modified: PropTypes.string,
+  changed_on: PropTypes.number.isRequired,
+  modified: PropTypes.string.isRequired,
```

Comment:
out of curiosity, why are these both not the same type? that seems more ideal.

Reply:
changed_on is timestamp number, modified is human readable string like `a month ago`, which is server-side provided convert timestamp to readable text.